x/accesscontrol/types/message_dependency_mapping.go

```diff
@@ -108,8 +122,25 @@ func DefaultWasmDependencyMappings() []acltypes.WasmDependencyMapping {
 	return []acltypes.WasmDependencyMapping{}
 }
 
+func MergeWasmAccessOps(accessOps1 []*acltypes.WasmAccessOperation, accessOps2 []*acltypes.WasmAccessOperation) []*acltypes.WasmAccessOperation {
+	if accessOps1 == nil || len(accessOps1) == 0 {
```

Comment:
the nil check isn't necessary right? since we're taking in an array of pointers instead of a pointer to an array of pointers?

Reply:
removed this func in favor of the new set usage